src/sentry/sentry_metrics/consumers/indexer/multiprocess.py

```diff
@@ -76,6 +76,7 @@ def submit(self, message: Message[Union[KafkaPayload, FilteredPayload]]) -> None
             # twice to snuba
             # TODO: Use the arroyo producer which handles FilteredPayload elegantly
             return
+
```

Comment:
nit: better to avoid touching this file at all

Reply:
Ooops, fixed. Thanks!